node/src/effect/announcements.rs

```diff
@@ -123,6 +124,15 @@ pub enum ConsensusAnnouncement {
     Finalized(Box<FinalizedBlock>),
     /// A linear chain block has been handled.
     Handled(Box<BlockHeader>),
+    /// An equivocation has been detected.
+    Equivocation {
+        /// The Id of the era in which the equivocation was detected
+        era_id: EraId,
+        /// The publickey of the equivocator.
```

Comment:
```suggestion
        /// The public key of the equivocator.
```

Reply:
@goral09 should this be changed to `Evidence` or `Fault` as well?

Reply:
If we change `Equivocation` to `Fault`, probably `equivocator` should change to `faulty validator`?

Reply:
`equivocation` is highway-specific, `evidence` sounds as if it was an actual _evidence_ which it isn't, `fault` is universal and it can be used for conflicting endorsements as well.

Reply:
Very well, I shall change it to `Fault`